instrumentation/opentelemetry-instrumentation-redis/tests/test_redis.py

```diff
@@ -78,6 +79,17 @@ def test_not_recording(self):
                 self.assertFalse(mock_span.set_attribute.called)
                 self.assertFalse(mock_span.set_status.called)
 
+    def test_suppress_instrumentation_no_span(self):
+        redis_client = redis.Redis()
+
+        token = context.attach(context.set_value(_SUPPRESS_INSTRUMENTATION_KEY, True))
```

Comment:
This should use the `supress_instrumentation` function. Check out [this example](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/a93bd74dc3969c13db1d6f603689ee8d9650ce66/instrumentation/opentelemetry-instrumentation-requests/tests/test_requests_integration.py#L399-L404).